Add_user.php

```diff
@@ -1,39 +1,74 @@
+<?php
+		// Deliver to login page if access to this page with out login
+		session_start();
+		if(!isset($_SESSION['login_user'])){
+				header("location:login.php");
+				}
+		else{
+			$User = $_SESSION['login_user'];
+		}
+?>
+
 <!DOCTYPE html>
 <html lang = "en">
 	<head>
 		<title>Add User</title>
 		<meta chrset="utf-8"/>
-		<meta name="Keywords" content = "html5, layout, CSS Grid System"/>
+		<meta name="Keywords" content = "html5"/>
 		<meta name = "Author" content = "Tony Nguyen"/>
-		<meta name = "Description" content = "CSS Grid System Layout Tutorial"/>
+		<meta name = "Description" content = "Music database"/>
 		<meta name = "viewport" content = "width=device-width. initial-scale=1"/>
-		
-		<link rel ="stylesheet" href = "test10.css"> 
+		<!--Connected to css -->
+		<link rel ="stylesheet" href = "test11.css"> 
 		
 	</head>
 <body>
+	<!--Set for grid system for the page -->
 	<div class ="grid-container">
+		<!-- The div for nav and in the div it contain the logo and a function to called the nav -->
 		<div class="navigation">
+			<!-- Set a div that contain logo for the music database -->
 			<div class="logo">
 				<a href = "01_login_real.php"><img src = "images/logo8.png" height = "100"></a>
 			</div>
 			<?php
-				//Pulls the links from the nav.php page and places them in the navigation div
+				//Pulls the links from the nav_admin.php page and places them in the navigation div
 				require 'nav_admin.php'; //'require' is 100% needed for the site to run 
 			?>
 		</div>
-		<div class="content"><!-- Holds the main page content -->
+		<div class="content"><!-- Holds the Add user form -->
 			<p>Add User</p>
-			<h3><form method = "post" id = "01_login">
+			<h3><form method = "post">
 			<input id="us" type = "text" name = "username" placeholder="User name"/><br />
 			<input id="pa" type = 'password' name = 'password' placeholder="Password"/><br />
 			<input  id="login" type = "submit" value = "Insert"/><br />
 			</form></h3>
+				<?php
```

Comment:
This is the php code that use to make them creating an account. If they was successfully create an account that it will be safe in the database and display will a text that is record inserted and if their account was invalid the it can't be safe in the database and it will display the text that say error inserting record.